src/ipc/ipc4/handler.c

```diff
@@ -32,6 +32,8 @@
 #include <user/trace.h>
 
```

Comment:
can we please add changes from handler.c and helper.c in 1 or 2 separate patches. 

remember, one patch should do one single logical change. Unless there is an entire new feature added. 

You have added the lib manager new feature in 1 patch, but the usage of it, bugfixes, helpers must come in separate patches with proper commit description.